keyboards/planck/keymaps/jetpacktuxedo/keymap.c

```diff
@@ -0,0 +1,287 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
```

Comment:
Could you replace this include with `#include QMK_KEYBOARD_H` instead?

Reply:
Hmmm... so I got the other fixes you asked for, but this one gets me
```
Compiling: keyboards/planck/keymaps/jetpacktuxedo/keymap.c                                         keyboards/planck/keymaps/jetpacktuxedo/keymap.c:17:26: fatal error: QMK_KEYBOARD_H: No such file or directory
 #include "QMK_KEYBOARD_H"
                          ^
compilation terminated.
```

Reply:
It shouldn't be in quotes.  it should be: `#include QMK_KEYBOARD_H`

Reply:
Oh, yep, that did it! Misled by my syntax highlighting...